sunpy/map/mapbase.py

```diff
@@ -1202,6 +1202,8 @@ def resample(self, dimensions: u.pixel, method='linear'):
         lon, lat = self._get_lon_lat(self.center.frame)
         new_meta['crval1'] = lon.value
         new_meta['crval2'] = lat.value
+        new_meta['naxis1'] = new_data.shape[1]
+        new_meta['naxis2'] = new_data.shape[0]
```

Comment:
There might be a rotate test that can be updated to check that this is set.

Reply:
ahhh yes, i was loking for it, but didnt see the one that tests metadata.

https://github.com/sunpy/sunpy/blob/6e40e8edce7fe0e75753620417fb51452db1d5e0/sunpy/map/tests/test_mapbase.py#L579

I will add it.